platforms/common/include/openmm/common/NonbondedUtilities.h

```diff
@@ -69,8 +69,10 @@ class OPENMM_EXPORT_COMMON NonbondedUtilities {
      * @param exclusionList  for each atom, specifies the list of other atoms whose interactions should be excluded
      * @param kernel         the code to evaluate the interaction
      * @param forceGroup     the force group in which the interaction should be calculated
+     * @param useNeighborList  specifies whether a neighbor list should be used to optimize this interaction.  This is
+     *                         should be viewed as only a suggestion.  Even when it is false, a neighbor list be used anyway.
```

Comment:
"This is should be" -> "This should be"

Reply:
Same with the other places where the typo was copied/pasted.

Reply:
"a neighbor list be used anyway." -> "a neighbor list _will_/_may_/_might_ be used anyway."

Reply:
More concise for the first sentence: "This should only be viewed as a suggestion."